### PR TITLE
Feat/add reversal android

### PR DESCRIPTION
**Pull request by JoshM1994**
* Adds the `reverse` and `boomerang` abilities to Android

* iOS equivalents to follow later today

### CODE REVIEW

File: lib/ProcessingManager/ProcessingManager.android.js [hunk 1]
```diff
@@ -49,6 +49,16 @@ export class ProcessingManager {
     return TrimmerManager.compress(actualSource, options);
   }
 
+  static boomerang(source: sourceType, options: any): Promise<*> {
```
Comment (shahen94) [inline lib/ProcessingManager/ProcessingManager.android.js]: Can you please specify options type?

File: lib/ProcessingManager/ProcessingManager.android.js [hunk 2]
```diff
@@ -49,6 +49,16 @@ export class ProcessingManager {
...
+    const actualSource: string = getActualSource(source);
+    return TrimmerManager.boomerang(actualSource, options);
+  }
+
+  static reverse(source: sourceType, options: any): Promise<*> {
```
Comment (shahen94) [inline lib/ProcessingManager/ProcessingManager.android.js]: Here as well

**Comment by JoshM1994** [comment]
Thanks for the feedback @shahen94 

- Added example usage to the README
- Removed the options entirely since they are not used
- Modified the return promise to be consistent with `trim` and `crop`

**Comment by shahen94** [comment]
@JoshM1994 You're awesome. Thanks !

**Pull request merged**